components/AppShell.vue

```diff
@@ -63,7 +63,6 @@
                       :href="item.href"
                       :target="item.target"
                       class="group mt-2 w-full flex items-center pl-2 pr-1 py-2 text-left text-content-primary text-base leading-6 font-medium rounded-md hover:bg-gray-200 focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
-                      :class="item.current ? 'bg-gray-100': ''"
```

Comment:
I've removed the shading here for now as it looks a bit strange on mobile view
![image](https://user-images.githubusercontent.com/46909693/213560960-f584d742-1c4f-4eeb-8465-b0571981a4a4.png)

Our mobile view design will improve in the future where these top-level elements will have their own disclosure
![image](https://user-images.githubusercontent.com/46909693/213561384-4f30ad45-3472-451a-baff-c50dd5bcf05c.png)